CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 - Make user segment a top level property ([#2257](https://github.com/getsentry/sentry-java/pull/2257))
+- Make user segment a top level property ([#2258](https://github.com/getsentry/sentry-java/pull/2258))
```

Comment:
PR title and changelog seem to be off, this is not about the `user.segment`.

Reply:
Oh wow good catch thanks! Not sure how that happened. Guess I just rely on the auto generated title too much and in this case it used the title from the PR I want to merge into.